config/routes.js

```diff
@@ -135,6 +135,9 @@ module.exports.routes = {
   'GET    /noo/logout':                                   'SessionController.destroy',
   'DELETE /noo/session':                                  'SessionController.destroySession',
 
+  'POST   /noo/token':                                    'TokenController.create',
```

Comment:
let's also use the term `access_token` or `auth_token` in the routes here and in the controller name